File: Practical10/app/src/main/java/com/example/practical10/Note.java
package com.example.practical10;


import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity(tableName = "notes")
public class Note {
    @PrimaryKey
    int id;
    private String title;
    private String content;

    public Note() {
    }

    public Note(int id, String title, String content) {
        this.id = id;
        this.title = title;
        this.content = content;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }
}

File: KotlinPractical12/app/src/main/java/com/example/kotlinpractical12/ui/data/Note.kt
package com.example.kotlinpractical12.ui.data

import androidx.room.Entity
import androidx.room.PrimaryKey


@Entity(tableName = "note")
data class Note(var title: String, var content: String) {

    @PrimaryKey(autoGenerate = true)
    var id: Int = 0

}




